FLEDGE.md

```diff
@@ -736,14 +741,78 @@ For detailed specification and explainers of the trusted key-value server, see a
 - [FLEDGE Key/Value Server APIs Explainer](https://github.com/WICG/turtledove/blob/master/FLEDGE_Key_Value_Server_API.md)
 - [FLEDGE Key/Value Server Trust Model Explainer](https://github.com/privacysandbox/fledge-docs/blob/main/key_value_service_trust_model.md)
 
+##### 3.1.1 Cross-Origin Trusted Server Signals
+
+If the key-value server is on a different origin than the corresponding script, additional steps are
+needed to ensure that sensitive information is not inappropriately leaked to, or is manipulated by,
+a third party.
+
+In case of bidder signals, the following changes occur:
+1. The fetch is a cross-origin CORS fetch with `Origin:` set to the buyer script's origin.
+2. The value is passed to `crossOriginTrustedSignals` parameter, not the `trustedBiddingSignals`
+   parameter, and there is one more level of nesting denoting the server's origin, e.g:
```

Comment:
question: Is there a benefit to this level of nesting, rather than simply another parameter indicating the origin / domain of the scoring signals?

Reply:
It's a bit harder to forget to check? (And also less parameters)

Reply:
>harder to forget to check? 

One is already explicitly checking the alternate signals parameter so that should already be priced in, so to speak.

Reply:
I was thinking in terms of having code do stuff like:
```
trustedSignals = trustedBiddingSignals || crossOriginTrustedSignals["kv-server.mydomain.com"];
```
...which simply can't access  an unexpected domain.